dietpi/dietpi-software

```diff
@@ -11615,6 +11655,62 @@ _EOF_
 			G_EXEC chown -R ffsync:ffsync /mnt/dietpi_userdata/firefox-sync
 
 		fi
+		
+                software_id=183 # Bitwarden RS
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+                    Banner_Configuration
+
+                    Create_User -d /mnt/dietpi_userdata/bitwarden_rs bitwarden_rs
+
+                    # Make dir and data dir needed to run
+                    G_EXEC mkdir -p /mnt/dietpi_userdata/bitwarden_rs/data
+
+                    # Make and edit the template config to have no web vault
+                    G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+                    G_EXEC sed -i 's/# WEB_VAULT_ENABLED=true/WEB_VAULT_ENABLED=false/g' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
+
+                    # Permissions
+                    G_EXEC chown -R bitwarden_rs:bitwarden_rs /mnt/dietpi_userdata/bitwarden_rs
+                    G_EXEC chmod +x /opt/bitwarden_rs/target/release/bitwarden_rs
+
+                    # Systemd service
+                    cat << _EOF_ > /etc/systemd/system/bitwarden_rs.service
+[Unit]
+Description=Bitwarden Server (Rust Edition)
+Documentation=https://github.com/dani-garcia/bitwarden_rs
+
+After=dietpi-boot.service network.target
+
+
+[Service]
+# The user/group bitwarden_rs is run under. the working directory (see below) should allow write and read access to this user/group
+User=bitwarden_rs
+Group=bitwarden_rs
```

Comment:
Let's do not define the group here. It is obsolete when `bitwarden_rs` is the primary group of that user, if breaks supplementary group permissions, e.g. adding it to an `ssl` group to grant access to SSL certificates, and if it's not the primary group, then it has been intentionally changed by admin, hence should be respected.

Reply:
OK, I half get what that means. The service I just copied and removed a few comments not applicable due to us using SQLite and not others